cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/setup/CheckImageProgress.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.cloud.event.setup;
```

Comment:
This is not an event, tehrefore it shall be placed into: com.sequenceiq.cloudbreak.cloud.model